lib/ini.js

```diff
@@ -104,16 +123,26 @@ function splitSections (str, separator) {
 }
 
 const decode = (str, opt = {}) => {
+  // The `typeof` check is required because accessing the `process` directly fails on browsers.
+  /* istanbul ignore next */
+  opt.platform = opt.platform || (typeof process !== 'undefined' && process.platform)
```

Comment:
I can't think of a compelling reason not to use [`os.EOL`](https://nodejs.org/api/os.html#oseol) instead of adding a new option here

Reply:
Upon further review this doesn't currently require *any* node modules, so `require('os')` would be a burden to existing browser users.

I also see this is copied from `encode`.  This is ok then.